src/features/calendar/components/EventShiftModal/EventModalTypeAutocomplete.tsx

```diff
@@ -38,8 +38,10 @@ const EventModalTypeAutocomplete: FC<EventModalTypeAutocompleteProps> = ({
     //In here, when the length of the type changes,
     //it searches for the created event and updates event with an ID.
     if (createdType !== '') {
-      const newId = types.find((item) => item.title === createdType)!.id;
```

Comment:
This (`!`) is actually not a [type assertion](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions), but a [non-null assertion](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-). They're very closely related of course, but I thought you might be curious to know the separate terms.

Reply:
ooh interesting!